tox.ini

```diff
@@ -40,6 +40,7 @@ commands=
            --disable=C,R \
            --disable=fixme \
            --disable=locally-disabled \
+           --disable=unsubscriptable-object \
```

Comment:
I would still add a comment explaining this is a temporary workaround, the issue is reported here: https://github.com/PyCQA/pylint/issues/3760